explainer.md

```diff
@@ -21,19 +21,18 @@ The following web application declares in its manifest that it can handle CSV an
 ```json
     {
       "name": "Grafr",
-      "file_handler": {
-        "action": "/open-files",
-        "files": [
-          {
-            "name": "raw",
-            "accept": [".csv", "text/csv"]
-          },
-          {
-            "name": "graph",
-            "accept": [".svg", "image/svg+xml"]
-          }
-        ]
-      }
+      "file_handler": [
+        {
+          "action": "/open-csv",
+          "name": "Raw Graph",
+          "accept": [ ".csv", "text/csv" ]
+        }
+        {
+          "action": "/open-svg",
+          "name": "SVG Graph",
+          "accept": [ ".svg", "text/svg+xml" ]
+        }
+      ]
     }
 ```
 
```

Comment:
When you say "action **MUST** be inside the app scope", drop the bold/caps for "MUST".

a) RFC2119 words (e.g. "**MUST**") are requirements imposed on conforming implementations (i.e., user agents), not on the documents being processed. In a spec, we wouldn't say that a document "**MUST**" do something, instead we would specify what the UA should do in the error cases, such as ignoring the file handler, reporting an error to the user, etc.
b) We should probably avoid using those terms in explainers anyway because they have a specific technical meaning which we don't need to be precise about here (it's like using WebIDL or other spec language; just explain things in English, and English does not include RFC2119 terms). That is, instead of rewording it to say what the UA should do in all cases, the explainer should just use the plain English word "must" instead of the RFC2119 word "**MUST**".

This comment was pretty musty.

Reply:
Done